BUILD.gn

```diff
@@ -219,6 +220,30 @@ run_node("bundle") {
   ]
 }
 
+run_node("bundle_compiler") {
```

Comment:
There's a bit of duplicated code with these two bundle commands. I wonder if this could be DRYed up by adding at template in [deno.gni](https://github.com/denoland/deno/blob/master/libdeno/deno.gni) ?